### DISCUSSION TITLE
Disconnect vpn OpenVpnServer not recevie message

### DISCUSSION THREAD

**Discussion by tt2nn** [schwabe/ics-openvpn#1606] [Configuration problems] [answered]
I use OpenVPN Connect app test VPN Server，wen i disconnect vpn，the OpenVpnServer can receive message
like: 
Apr 12 08:25:38 openvpn-node-test-uk-a-1 openvpn[3885]: first client/123.113.187.160:9097 SIGTERM[soft,remote-exit] received, client-instance exiting

Then i use OpenVPN Remote demo test VPN Server, Wen i disconnect vpn, the OpenVpnServer can not receive any message.
after about 5 min, can get timeout message.

How can i do that , thanks

**Comment by schwabe** [comment, marked as answer]
This seems more a configuration issue of using explicit-exit-notify

**Reply by tt2nn** [reply]
it worked
Thanks
